public/js/test.js

```diff
@@ -60,6 +60,15 @@ function removeUser() {
 //   }
 // }
 
+function removeLoader(){
+  if(document.getElementsByClassName("input-group-button")[0].classList.contains("loading-data")){
```

Comment:
Do we need to add any checks that `document.getElementsByClassName("input-group-button")` is non-null before trying to access the `[0]` index?

Also, not sure if we want to try caching the "input-group-button" value so we don't need to call it 2-3 times.

Reply:
Just pushed with the event listener wrapped in a conditional that checks for the form but I could add another conditional here if you think that would be better.